twilio/base/client_base.py

```diff
@@ -132,9 +146,19 @@ async def request_async(
                 "http_client must be asynchronous to support async API requests"
             )
 
-        auth = self.get_auth(auth)
+
         headers = self.get_headers(method, headers)
-        uri = self.get_hostname(uri)
```

Comment:
uri should be there in updated version as well right?

Reply:
Updated